dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/BinaryEncoding.java

```diff
@@ -61,4 +61,37 @@ public String getContentType() {
     public boolean supportsContentType(String contentType) {
         return Encodings.matchesContentType(CONTENT_TYPE, contentType);
     }
+
+    @Override
+    public String toString() {
+        return "BinaryEncoding{" + CONTENT_TYPE + '}';
+    }
+
+    enum OptionalInputStreamDeserializer implements Deserializer<Object> {
+        INSTANCE;
+
+        @Override
+        public Object deserialize(InputStream input) {
+            return Optional.of(input);
+        }
+
+        @Override
+        public String toString() {
+            return "OptionalInputStreamDeserializer{}";
+        }
+    }
+
+    enum InputStreamDeserializer implements Deserializer<Object> {
+        INSTANCE;
+
+        @Override
+        public Object deserialize(InputStream input) {
+            return input;
+        }
+
+        @Override
+        public String toString() {
+            return "InputStreamDeserializer{}";
```

Comment:
I think the sls mapper will use the name (INSTANCE) since this is an enum, this is helpful for the debugger though